### ISSUE TITLE
Missing artifact into maven central repository

### ISSUE THREAD

**Issue by paissad** [Inform-Software/sonar-groovy#18]
Hello,

i don't see the versions 1.5 and 1.6 into the maven central repository https://repo1.maven.org/maven2/org/sonarsource/groovy/sonar-groovy-plugin/ 
Is it planned to add them ?

i'm using SonarQube 7.8 , and if i understand the table correctly, i should use version 1.6 of the groovy plugin

Regards,

**Comment by TobiX** [comment]
No, since you can install this plugin from the SonarQube marketplace, it doesn't need to be on Maven Central

**Comment by paissad** [comment]
@TobiX  actually, i'm developing a plugin which relies on sonar-xml and sonar-groovy, ... i added sonar-xml as a dependency (from maven central), but i could not add the latest version of sonar-groovy as a dependencey since only the version 1.4 is available so far.
Downloading sonar-groovy from marketplace and installing it in SonarQube is not the real concern here. Developing a plugin by using the correct version is my main concern.
I hope you understand my use a bit better now.

**Comment by TobiX** [comment]
Fair enough. Would BinTray be enough? AFAIR deploying to Maven Central is quite a process... Using BinTray is just a bunch of clicks :smile:

**Comment by paissad** [comment]
@TobiX I don't know if bintray is globally accessible from the artifacory repository i use.
For me, if you already have your maven central account, then it's pretty fast to upload an artifact.
Do you have some artifacts in your bintray repository so i can check if it's OK for accessing it ?

**Comment by TobiX** [comment]
I added a BinTray repository for this plugin. Release artifacts can be downloaded from there:

- Web: https://bintray.com/tobix/sonar-groovy/sonar-groovy
- Maven repo: https://dl.bintray.com/tobix/sonar-groovy

Future artifacts will be uploaded on release. I retroactively uploaded 1.6 using Maven, so the artifacts are not bit-identical to that release, even if they contain the same code.

Depending on the plugin should be the same as before:
```xml
<dependency>
  <groupId>org.sonarsource.groovy</groupId>
  <artifactId>sonar-groovy-plugin</artifactId>
  <version>1.6</version>
</dependency>
```

**Comment by TobiX** [comment]
Since BinTray has been deprecated, new releases will be deployed to GitHub packages: https://github.com/Inform-Software/sonar-groovy/packages/800725

You probably need to add the repository somehow and then add:

```
 <dependency>
  <groupId>org.sonarsource.groovy</groupId>
  <artifactId>sonar-groovy-plugin</artifactId>
  <version>1.7</version>
</dependency>
```
